site/src/pages/WorkspacePage/WorkspacePage.test.tsx

```diff
@@ -113,7 +113,8 @@ describe("WorkspacePage", () => {
     const confirmButton = await screen.findByRole("button", { name: "Delete" })
     await user.click(confirmButton)
     expect(deleteWorkspaceMock).toBeCalled()
-  })
+    // This test takes long to finish
```

Comment:
nit: we should add a TODO here to look into this later

Reply:
Maybe an issue instead of a TODO

Reply:
Not sure if it is needed. The workspace page test is doing a lot of renders so I think this is justified. Probably, we can move or remove some of them if we get a different strategy.